.github/workflows/auth-test.yml

```diff
@@ -0,0 +1,37 @@
+name: 'Test gcloud authentication'
+
+on:
+  pull-request:
+    branches:
+    - main
+
+permissions: read-all
+
+jobs:
+  terraform:
+    name: 'gcloud authentication test'
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        shell: bash
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+    - name: Setup Environment
```

Comment:
I think this step is no longer needed as well.

Reply:
It's needed if we use the exported credentials file (see [here](https://github.com/google-github-actions/auth#other-inputs under `create_credentials_file`)

Can't tell if it's used though

Reply:
`create_credentials_file` defaults to true, and it exports `GOOGLE_APPLICATION_CREDENTIALS` for us automatically right?

Reply:
Yes, provided the checkout action is used above

Reply:
Oh sorry, I forgot to push my latest change. I thought you were talking about the checkout action!